### ISSUE TITLE
Feature Extraction model with 10% of the training data

### ISSUE THREAD

**Issue by Rhino-byte** [Rhino-byte/Food-Image-classification#4]
We will train the Food 101 dataset with various pre-trained models from TensorFlow efficientNetBX model to find the best-performing model that we will use for fine-tuning our data

**Comment by Rhino-byte** [comment]
so  far so good most of the efficient net models have a good start off with an accuracy of 69-70% results for more on this check out [Weights and biases]( https://api.wandb.ai/links/bushman254/3czkmu5s)

**Comment by Rhino-byte** [comment]
![Image](https://github.com/user-attachments/assets/58d82413-acea-4273-86af-82e313fb208a)

Using image augmentation (Random_zoom ,Random_Height, Random_flip(horizontal =True))
